modules/rostests/winetests/ws2_32/sock.c

```diff
@@ -7008,6 +7008,11 @@ static void test_write_watch(void)
     size = 0x10000;
     base = VirtualAlloc( 0, size, MEM_RESERVE | MEM_COMMIT | MEM_WRITE_WATCH, PAGE_READWRITE );
     ok( base != NULL, "VirtualAlloc failed %u\n", GetLastError() );
+    if (base == NULL)
+    {
+        skip("failed to allocate %u bytes. Skipping test.", size);
```

Comment:
```suggestion
        skip("Failed to allocate %u bytes. Skipping test.", size);
```
How frequently do we get this allocation failing?

Reply:
It always fails, because we do not support MEM_WRITE_WATCH. I decided to revert the change for now though, because it only makes it worse: instead of crashing, the test hangs later.